ocaml/idl/datamodel_pool.ml

```diff
@@ -1023,6 +1035,52 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let install_rpmgpgkey =
+  call ~name:"install_rpmgpgkey"
+    ~doc:"Install an RPM GPG public key onto the pool."
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; (String, "name", "The name of the RPM GPG public key")
+      ; (String, "pubkey", "The RPM GPG public key")
+      ]
+    ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
+    ~lifecycle:[]
+    ~result:(Ref _gpg_key, "The reference of the created Gpg_key object")
+    ()
+
+let uninstall_rpmgpgkey =
+  call ~name:"uninstall_rpmgpgkey"
+    ~doc:"Uninstall an RPM GPG public key from the pool."
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; (String, "name", "The name of the RPM GPG public key to be uninstalled")
```

Comment:
This argument makes the implementation more difficult than it should be, use `(Ref _gpg_key, "gpg_key", "...")` instead